src/Misc/externals.sh

```diff
@@ -25,7 +25,7 @@ EXTERNALTOOLSLOCATION=(
     portableredhatgit/1
     portablewingit/1
     symstore/1
-    tee/1
+    tee/14_0_4
```

Comment:
It would be nice to switch to a meaningful version number for externals like TEE (other products). Thoughts?